pkg/operation/seed/seed.go

```diff
@@ -323,6 +326,20 @@ func BootstrapCluster(seed *Seed, secrets map[string]*corev1.Secret, imageVector
 		sgFluentdCredentials := deployedSecretsMap["fluentd-es-sg-credentials"]
 		sgFluentdPassword = string(sgFluentdCredentials.Data[utilsecrets.DataKeyPassword])
 		sgFluentdPasswordHash = string(sgFluentdCredentials.Data[utilsecrets.DataKeyPasswordBcryptHash])
+
+		//Read extension provider-specific configuration
+		existingConfigMaps := &corev1.ConfigMapList{}
+		if err = k8sSeedClient.Client().List(context.TODO(), existingConfigMaps, client.InNamespace(common.GardenNamespace)); err != nil {
+			return err
+		}
+
+		// Read all filters and parsers comming from the extension provider configurations
+		for _, cm := range existingConfigMaps.Items {
+			if strings.HasPrefix(cm.Name, "gardener-extension-provider-") && strings.HasSuffix(cm.Name, "-logging-config") {
+				filters.WriteString(fmt.Sprintln(cm.Data["filter-kubernetes.conf"]))
```

Comment:
Do we have to handle the cases where "filter-kubernetes.conf" and/or "parsers.conf" (create constants) are not present?

Reply:
I don't think so. No configurations from the extension provider will result in an empty row in the `fluent-bit` configmap.

Reply:
OK, thank you.